e2e/scripts/pageHelpers.js

```diff
@@ -0,0 +1,365 @@
+const { expect } = require("@playwright/test");
+const config = require("../config");
+const { getCreditCardExpiry } = require("../scripts/utils.js")
+
+/**
```

Comment:
Thanks a lot for cleaning this up into helper methods. Looks a lot cleaner now :D